src/test/lambda/local/detectLocalTemplates.test.ts

```diff
@@ -108,42 +65,9 @@ describe('detectLocalTemplates', async () => {
         assert.strictEqual(templates[0].fsPath, normalizePath(workspaceFolderChildPath, 'template.yaml'))
     })
 
+    // TODO delete this test when this check is fixed
     it('Does not recursively descend past the direct children of each workspace folder', async () => {
-        const workspaceFolderPath = normalizePath('my', 'workspace', 'folder')
```

Comment:
I think this test will never fail in its current form.
Looks like this test needs to create the child, grandchild folders, and write the yaml file to the grandchild folder.

Reply:
Fixed